Main.elm

```diff
@@ -14,7 +14,11 @@ import List.Extra
 
 
 type alias Model =
-  List ( String, String )
+  List Param
+
+
+type alias Param =
+  ( String, String )
```

Comment:
Doing these changes means everything is a bit more self documenting and if the concept if a `Param` changes we only have to change one type defn.